pkg/scheduler/cache/cache.go

```diff
@@ -457,9 +456,9 @@ func (sc *SchedulerCache) setBatchBindParallel() {
 
 func (sc *SchedulerCache) setDefaultVolumeBinder() {
 	logger := klog.FromContext(context.TODO())
-	var capacityCheck *volumescheduling.CapacityCheck
+	var capacityCheck volumescheduling.CapacityCheck
 	if options.ServerOpts != nil && options.ServerOpts.EnableCSIStorage && utilfeature.DefaultFeatureGate.Enabled(features.CSIStorage) {
-		capacityCheck = &volumescheduling.CapacityCheck{
```

Comment:
Why this changed?

Reply:
because  the type of formal parameter named capacityCheck of function volumescheduling.NewVolumeBinder has be changed from pointer to  struct. And the variable capacityCheck  be used in setDefaultVolumeBinder method only. So I try to changing it.